tests/mir-opt/copy-prop/dead_stores_better.rs

```diff
@@ -1,4 +1,3 @@
-// skip-filecheck
 // EMIT_MIR_FOR_EACH_PANIC_STRATEGY
 // This is a copy of the `dead_stores_79191` test, except that we turn on DSE. This demonstrates
 // that that pass enables this one to do more optimizations.
```

Comment:
The compile-flags are not up-to-date. They should specify `DeadStoreElimination-initial`

Reply:
Modified in https://github.com/rust-lang/rust/pull/135099/commits/0d6fa54c231850af21adb9fe8087af180798e682